### PR TITLE
EPMRPP-34183 Pagination toolbar hides when no filters

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

**Comment by ArtsiomYeliseyenka** [comment]
Please resolve conflicts. Moreover, to avoid that situation in future, before each push, please do fetch+rebase onto.

File: app/src/pages/inside/filtersPage/filterTable/filterTable.jsx [hunk 1]
```diff
@@ -88,16 +88,19 @@ export class FilterTable extends PureComponent {
...
-            activePage={activePage}
-            itemCount={itemCount}
-            pageCount={pageCount}
-            pageSize={pageSize}
-            onChangePage={onChangePage}
-            onChangePageSize={onChangePageSize}
-          />
-        </div>
+        {data &&
+          !!data.length && (
```
Comment (ArtsiomYeliseyenka) [inline app/src/pages/inside/filtersPage/filterTable/filterTable.jsx]: !! unnecessary here

Comment (AmsterGet) [reply app/src/pages/inside/filtersPage/filterTable/filterTable.jsx]: So the same structure exists in the line before in this component
Can I replace it there with data.length too?
image

Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/filtersPage/filterTable/filterTable.jsx]: yeah

Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/filtersPage/filterTable/filterTable.jsx]: hmm, let it be. Sorry for my fallacy

File: app/src/pages/inside/filtersPage/filterTable/filterTable.jsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ export class FilterTable extends PureComponent {
       <Fragment>
         <div className={cx('filter-table')}>
           <FilterTableHeader />
-          {data && !!data.length ? (
+          {data && data.length ? (
```
Comment (DiscoElevator) [inline app/src/pages/inside/filtersPage/filterTable/filterTable.jsx]: `data && !!data.length`

**Comment by DiscoElevator** [comment]
Please resolve conflicts

**Pull request merged**